spec/core_functions/color/is_legacy.hrx

```diff
@@ -0,0 +1,284 @@
+<===> prophoto-rgb/input.scss
+@use "sass:color";
+a {b: color.is-legacy(color(prophoto-rgb 0.42444 0.934918 0.190922))}
+
+<===> prophoto-rgb/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> rec2020/input.scss
+@use "sass:color";
+a {b: color.is-legacy(?????)}
+
+<===> rec2020/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> xyz/input.scss
+@use "sass:color";
+a {b: color.is-legacy(color(xyz 0.0426 0.0442 0.0364))}
+
+<===> xyz/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> xyz-d50/input.scss
+@use "sass:color";
+a {b: color.is-legacy(color(xyz-d50 0.2005 0.14089 0.4472))}
+
+<===> xyz-d50/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> xyz-d65/input.scss
+@use "sass:color";
+a {b: color.is-legacy(color(xyz-d65 0.21661 0.14602 0.59452))}
+
+<===> xyz-d65/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> lab/input.scss
+@use "sass:color";
+a {b: color.is-legacy(lab(5 110 115))}
+
+<===> lab/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> lch/input.scss
+@use "sass:color";
+a {b: color.is-legacy(lch(90.6 52.8 197))}
+
+<===> lch/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> oklab/input.scss
+@use "sass:color";
+a {b: color.is-legacy(oklab(0.44027 0.08818 -0.13386))}
+
+<===> oklab/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> oklch/input.scss
+@use "sass:color";
+a {b: color.is-legacy(oklch(70% 0.1 200))}
+
+<===> oklch/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> error/too_few_args/input.scss
+a {b: color.is-legacy(rgb(0 255))}
+
+<===> error/too_few_args/error
+Error: Missing argument $amount.
+  ,--> input.scss
+1 | a {b: color.is-legacy(rgb(0 255))}
+  |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^ invocation
+  '
+  ,--> sass:color
+1 | @function color.is-legacy(rgb($amount $amount $amount)) {
+  |           ============================================= declaration
+  '
+  input.scss 1:7  root stylesheet
+
+<===> error/too_few_args/error-libsass
+Error: Function color.is-legacy is missing argument $amount.
+        on line 1 of input.scss
+>> a {b: color.is-legacy(rgb(0 255))}
+
+   ------^
+
+<===>
+================================================================================
+<===> error/too_many_args/input.scss
+a {b: color.is-legacy(rgb(0 255 0 15))}
```

Comment:
Same here. We're just trying to test `is-legacy()`. A "too many" args error should be triggered if we run this function with more than one color:

`a {b: color.is-legacy(rgb(0 255 0), blue)}`

(see the functions for blue() and blackness() for examples of errors for a single argument function)